vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_header.scss

```diff
@@ -8,7 +8,7 @@ header {
 	height: 80px;
 	padding: 26px 50px;
 	box-shadow: 0 1px 11.5px 0 rgba($black, 0.18);
-  	@include respond-above(sm) {
+  	@include respond-above(md) {
```

Comment:
@FionaDL This PR is supposed to create new breakpoints, right? Because it's actually replacing the existing ones for `sm` views. Shouldn't we keep these breakpoints and add new ones? What happens with the small devices?

Reply:
@cleicar Hmmm, when we change it to `md` instead of `sm` the style will apply to both md and sm. The problem in fastruby.io is that the small breakpoint wasn't working for what we needed to happen. The medium break point allows the header to become a hamburger menu and hides the navbar when we reach medium instead of small.